### DISCUSSION TITLE
Delete Project

### DISCUSSION THREAD

**Discussion by veenkumaar** [deta/cloud-docs#514] [Help]
Hi, I'm new in Deta and i just created project and upload some data and it's works perfect.
When i run command in terminal "deta projects" i've seen two project first is default another is created by mistake. So, i would like to know how to delete another project in project list.

**Comment by Sokhavuth** [comment]


**Reply by Sokhavuth** [reply]
I thought you talking about Micro. Sorry!

**Comment by jnsougata** [comment]
> Hi, I'm new in Deta and i just created project and upload some data and it's works perfect. When i run command in terminal "deta projects" i've seen two project first is default another is created by mistake. So, i would like to know how to delete another project in project list.

Currently there's no way to delete any project. However a deta base or drive or micro inside a project can be deleted.

**Comment by lemonyte** [comment]
Remember to search for similar questions before asking 🙂

https://github.com/orgs/deta/discussions/459
